symfony/webpack-encore-bundle/1.0/package.json

```diff
@@ -8,9 +8,9 @@
     "license": "UNLICENSED",
     "private": true,
     "scripts": {
-        "dev-server": "encore dev-server",
-        "dev": "encore dev",
-        "watch": "encore dev --watch",
-        "build": "encore production --progress"
+        "dev-server": "NODE_ENV=development encore dev-server",
+        "dev": "NODE_ENV=development encore dev",
+        "watch": "NODE_ENV=development encore dev --watch",
+        "build": "NODE_ENV=production encore production --progress"
```

Comment:
how does this work when the var already exists?
should it be `NODE_ENV=${NODE_ENV:-production}`?

Reply:
What question me the most is: will this be compatible with all platforms?